lua/arrow/ui.lua

```diff
@@ -332,8 +332,7 @@ function M.openFile(fileNumber, previousFile)
 		end
 
 		closeMenu()
-
-		action(fileName, vim.b.filename)
+		action(config.getState('save_key')() .. '/' .. fileName, vim.b.filename)
```

Comment:
I'm uncertain how this would work on a non-Unix-based filesystem. I'm assuming that this is fine since the rest of the codebase lacks Windows path handling.